d2renderers/d2svg/d2svg.go

```diff
@@ -967,31 +1091,26 @@ func drawShape(writer io.Writer, targetShape d2target.Shape, sketchRunner *d2ske
 			// we need the self closing form in this svg/xhtml context
 			render = strings.ReplaceAll(render, "<hr>", "<hr />")
 
-			var mdStyle string
-			if targetShape.Fill != "" {
-				mdStyle = fmt.Sprintf("background-color:%s;", targetShape.Fill)
-			}
-			if targetShape.Stroke != "" {
-				mdStyle += fmt.Sprintf("color:%s;", targetShape.Stroke)
-			}
-
-			fmt.Fprintf(writer, `<div xmlns="http://www.w3.org/1999/xhtml" class="md" style="%s">%v</div>`, mdStyle, render)
+			mdEl := svgstyle.NewThemableElement("div")
+			mdEl.Xmlns = "http://www.w3.org/1999/xhtml"
```

Comment:
this doesn't need to be set every time, we should just default add it to every render of a `div`